data/moves.ts

```diff
@@ -16238,7 +16239,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 		secondary: {
 			dustproof: true,
 			chance: 100,
-			onHit(target) {
+			onHit(target, source, move) {
```

Comment:
```suggestion
			onHit(target) {
```
no need to add `source` and `move` when you're not using them (and when they weren't there in the first place)